.travis.yml

```diff
@@ -1 +1,9 @@
 language: crystal
+
+install:
+  - shards install
+
+script:
+  - crystal spec
+  - crystal tool format --check src spec
+  - bin/ameba src
```

Comment:
Why only `src`? Did you have issues inspecting files in `spec`?

Reply:
Yup, I had couple of them:

- False positives in first case - I'd say it's an issue:
  ```
  spec/spec_helper.cr:8:1
  RedundantBegin: Redundant `begin` block detected

  spec/spec_helper.cr:18:1
  RedundantBegin: Redundant `begin` block detected
  ```

- `LargeNumbers` rule in the second - just an annoyance:
  ```
  spec/raven/processors/sanitize_data_spec.cr:61:41
  LargeNumbers: Large numbers should be written with underscores: 123_456_789

  spec/raven/processors/sanitize_data_spec.cr:101:39
  LargeNumbers: Large numbers should be written with underscores: 123_456_789

  spec/raven/processors/sanitize_data_spec.cr:159:24
  LargeNumbers: Large numbers should be written with underscores: 4_242_424_242_424_242

  spec/raven/processors/sanitize_data_spec.cr:174:26
  LargeNumbers: Large numbers should be written with underscores: 4_242_424_242_424_242

  spec/raven/processors/sanitize_data_spec.cr:179:39
  LargeNumbers: Large numbers should be written with underscores: 4_242_424_242_424_242
  ```

Reply:
@Sija okay, thanks. Reported https://github.com/veelenga/ameba/issues/56.

`LargeNubers` works as it is designed. You can easily disable it if it is an annoyance for you:

```yml
# .ameba.yml
LineNumbers:
  Enabled: false
```